src/components/item/sharing/CustomizedTagsEdit.js

```diff
@@ -89,6 +95,10 @@ function CustomizedTagsEdit({ item, edit }) {
           {t('Save')}
         </Button>
       </form>
+      <Typography variant="subtitle1">{t('Tags Preview')}</Typography>
```

Comment:
Could you provide a screenshot? Maybe you don't need another title for that.

Reply:
![image](https://user-images.githubusercontent.com/25419619/147153093-2e33299f-8943-4126-9075-d5fd5dad6d39.png)